src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -255,7 +292,62 @@ describe('sidebar.components.sidebar-content', function() {
       assert.isTrue(updateSpy.calledWith(frameUris[1], true));
     });
 
-    context('when there is a selection', function() {
+    context('when there is a direct-linked group error', () => {
+      beforeEach(() => {
+        setFrames([{ uri: 'http://www.example.com' }]);
+        fakeSettings.group = 'group-id';
+        store.loadGroups([{ id: 'default-id' }]);
+        store.focusGroup('default-id');
+        fakeGroups.focused.returns({ id: 'default-id' });
+        $scope.$digest();
+        // Re-construct the controller after the environment setup.
+        makeSidebarContentController();
+      });
+
+      it('sets directLinkedGroupFetchFailed to true', () => {
+        assert.isTrue(ctrl.directLinkedGroupFetchFailed);
+      });
+
+      it('sets areAllAnnotationsVisible to true since there is an error message', () => {
```

Comment:
Similar to above, I think this test and the one below it would be slightly clearer if written as "{methodName} returns {value}", because `areAllAnnotationsVisible` and `selectedGroupUnavailable` just return a value rather than being a mutable property which gets set (unlike `directLinkGroupFetchFailed`, it makes sense that gets "set")